third_party/pyth/price-service/src/rest.ts

```diff
@@ -181,6 +180,51 @@ export class RestAPI {
       "api/latest_price_feeds?ids[]=<price_feed_id>&ids[]=<price_feed_id_2>&.."
     );
 
+    app.get(
+      "/api/latest_price_info",
+      validate(latestPriceFeedsInputSchema),
+      (req: Request, res: Response) => {
+        let priceIds = req.query.ids as string[];
+
+        let responseJson = [];
+
+        let notFoundIds: string[] = [];
+
+        for (let id of priceIds) {
+          if (id.startsWith("0x")) {
+            id = id.substring(2);
+          }
+
+          let latestPriceInfo = this.priceFeedVaaInfo.getLatestPriceInfo(id);
+
+          if (latestPriceInfo === undefined) {
+            notFoundIds.push(id);
+            continue;
+          }
+
+          const freshness: DurationInSec =
+            new Date().getTime() / 1000 - latestPriceInfo.attestationTime;
+          this.promClient?.addApiRequestsPriceFreshness(
+            req.path,
+            id,
+            freshness
+          );
+
+          responseJson.push({"price_feed": latestPriceInfo.priceFeed.toJson(), "emitter_chain": latestPriceInfo.emitterChainId, "attestation_time": latestPriceInfo.attestationTime, "sequence_number": latestPriceInfo.seqNum});
```

Comment:
it seems like this is the only line of code that changed from the method above ?

Can you make this an additional query parameter on the existing endpoint? I think (a) that would make the overall API less confusing to users, and (b) would reduce code duplication.

Reply:
that was the plan initially but I had a call with @ali-bahjati and he suggested that we use a separate endpoint rather than query params  -- but happy to implement query params as well

Reply:
yeah I suggested this...

I want us to have an explicit API for price feeds, unless we discuss we don't want it. We can stop exposing it to users (if it's consuming).

Wherever we expose `price_feed` I think it's better that the returned json to be consistent with our definition of our price feed, with no more info in it (unless it's added everywhere).

Although I'm fine with removing `get_latest_price_feeds` entirely or make it return price feed as a nested value and has the rest of the infos with query param (or they always be there).